src/ComponentTester.cpp

```diff
@@ -26,20 +31,17 @@ void ComponentTester::testPin(){
     tone(pin, 1000, 100); //(Buzzer)_component.tone(1000) wouldn't work. Polymorphism?
   }
   if(componentType.equalsIgnoreCase("PHOTORESISTOR")){
-    for(int i = 1; i <= 10; i++){
+    /*for(int i = 1; i <= 10; i++){
```

Comment:
So I'm having this error where incorrect parameters are being passed to my logger class.... The weirdest things break the code. I'll have to tell you about it tomorrow. This was one that was breaking the code randomly that I commented out but didn't uncomment. We can fix that.